Dockerfile

```diff
@@ -1,30 +1,123 @@
-FROM docker.mirror.hashicorp.services/ubuntu:16.04
+# This Dockerfile contains multiple targets.
+# Use 'docker build --target=<name> .' to build one.
+# e.g. `docker build --target=official-light .`
+#
+# All non-dev targets have a VERSION argument that must be provided 
+# via --build-arg=VERSION=<version> when building. 
+# e.g. --build-arg VERSION=1.11.2
+#
+# `default` is the production docker image which cannot be built locally. 
+# For local dev and testing purposes, please build and use the `dev` docker image.
 
-ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update && apt-get install -y \
-    locales \
-    openssh-server \
-    sudo
+# Development docker image
+# This image includes all source code found in this repository.
+# This is primarily used for development and debugging.
+FROM docker.mirror.hashicorp.services/alpine:latest as dev
```

Comment:
In the doc above, and in the comment just above this line, it says that the `dev` image is the same as `full` in that it contains all the source code. However, this target seems to just use an `ADD` to copy a prebuilt binary inside the image.

Reply:
The instructions on building the dev image are in the makefile. I can make these comments consistent so it's clear what's happening. Thanks! :)